pkg/controllers/edgeconnect/controller.go

```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/edgeconnect/deployment"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/edgeconnect/version"
 	"github.com/Dynatrace/dynatrace-operator/pkg/oci/registry"
-	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/hasher"
+	deployment2 "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/deployment"
```

Comment:
```suggestion
	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/deployment"
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/9d5f5bab246463ea2891ff91d9d1ee7096c84c05